Include/interactivity.h

```diff
@@ -237,6 +237,13 @@ class interactive_participant
 enum class interactive_event_type
 {
     /// <summary>
+    /// The message is not a message processed by the SDK. It could be a custom
+    /// message from an interactive control or a message supported by the protocol
+    /// but not yet supported by the SDK.
+    /// </summary>
+    unknown,

```

Comment:
I thought about doing "custom". "unknown" was accurate. "generic" didn't feel quite right either. But I agree unknown is scary

---
In reply to: [132342903](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132342903) [](ancestors = 132342903)